Mathlib/Algebra/Order/Archimedean/Class.lean

```diff
@@ -453,3 +457,112 @@ theorem liftOrderHom_mk (f : M → α) (h : ∀ a b, mk a ≤ mk b → f a ≤ f
 end LiftHom
 
 end MulArchimedeanClass
+
+variable (M) in
+/-- Subtype of `MulArchimedeanClass` that removes the class of `1` (i.e. the top element). -/
```

Comment:
```suggestion
/-- Subtype of `MulArchimedeanClass` that removes the class of `1` (i.e. the top element).

This is useful since the family of non-top archimedean classes is linearly independent.
-/
```
or something like that.

Also, I personally don't like docstrings that say "Same thing as `foo`, but we change bar", because it becomes difficult to understand at once. Actually, now that I reread your docstring for `MulArchimedeanClass`, it suffers the same issue! Could you try making both docstrings more self-contained?

Reply:
Updated the doc